torchdrivesim/behavior/iai.py

```diff
@@ -62,12 +129,15 @@ class IAIWrapper(NPCWrapper):
     """
     def __init__(self, simulator: SimulatorInterface, npc_mask: TensorPerAgentType,
                  recurrent_states: List[List], locations: List[str], rear_axis_offset: Optional[Tensor] = None,
+                 car_sequences: Optional[Dict[int, List[List[float]]]] = None,
```

Comment:
It's used by `torchdriveenv` to set `parked car`, and can also be used to load preset logs.

Potentially, it can be replaced by `ReplayWrapper` with careful design, but currently, let's leave it here.

Reply:
I can accept that, but can you at least make it `replay_states` (BxAxTxSt float) and `replay_mask` (BxA or BxAxT bool) so that we use tensors consistently, rather than resorting to ad-hoc formatted dictionaries?